usr/share/rear/lib/_input-output-functions.sh

```diff
@@ -403,6 +403,14 @@ function UserInput () {
     local timeout=300
     # Avoid stderr if USER_INPUT_TIMEOUT is not set or empty and ignore wrong USER_INPUT_TIMEOUT:
     test "$USER_INPUT_TIMEOUT" -ge 0 2>/dev/null && timeout=$USER_INPUT_TIMEOUT
+    # Have some seconds (at least one second) delay when an automated user input is used to be fail-safe against
+    # a possibly false specified predefined user input value for an endless retrying loop of UserInput calls
+    # that would (without the delay) run in a tight loop that wastes resources (CPU, diskspace, and memory)
+    # and fills up the ReaR log file (and the disk - which is a ramdisk for 'rear recover')
+    # with some KiB data each second that may let 'rear recover' fail with 'out of diskspace/memory':
+    local automated_input_interrupt_timeout=3
+    # Avoid stderr if USER_INPUT_INTERRUPT_TIMEOUT is not set or empty and ignore wrong USER_INPUT_INTERRUPT_TIMEOUT:
+    test "$USER_INPUT_INTERRUPT_TIMEOUT" -ge 1 2>/dev/null && automated_input_interrupt_timeout=$USER_INPUT_INTERRUPT_TIMEOUT
```

Comment:
What I mean is:
While I am replacing existing read and select user input
by the UserInput function for ReaR v 2.3 I assume I will
detect this or that special case that needs further changes
and then I would like to be free of backward compatibility.